### PR TITLE
Hyperledger Bevel Q3 report

### PR DESCRIPTION
**Pull request by suvajit-sarkar**

### CODE REVIEW

File: project-reports/2024/2024-Q3-Hyperledger-Bevel.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+- Continue with Q3 plan and release commitments.
+- Hyperledger meetup: Bevel workshop series remaining two sessions in September.
+- Explore cloud native vaults (AWS secret manager).
+- Collaborate with hyperledger-labs blockchain explorer and fabric-opertaions-console teams and plan to add deployment support on bevel.
+
+# Maintainer Diversity
+
+-	There is change in the maintainers for Bevel. You can find our current maintainer list here:  
+<a href="https://github.com/hyperledger/bevel/blob/main/MAINTAINERS.md" class="external-link" rel="nofollow">https://github.com/hyperledger/bevel/blob/main/MAINTAINERS.md</a>.
```
Comment (arsulegai) [inline project-reports/2024/2024-Q3-Hyperledger-Bevel.md:56]: The maintainers file contents does not match the list in this report.

File: project-reports/2024/2024-Q3-Hyperledger-Bevel.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+# 2024 Q3 Hyperledger Bevel
+
+# Project Health
+
+[Hyperledger Bevel LFX Insights page](https://insights.lfx.linuxfoundation.org/foundation/hyp/overview?project=bevel&repository=&dateFilters=Last%20Year&dateRange=2023-01-01%20to%202023-12-31&compare=PP&granularity=month&hideBots=true)
+
+
+1. Getting code contributions for Hyperledger Bevel still remains a challenge.
+2. The Bevel project has received no external contributions; nearly all contributions for the last couple of releases have come from the Accenture team. Accenture will not be able to maintain the project without other organization or individual contributor help in maintaining.
```
Comment (arsulegai) [inline project-reports/2024/2024-Q3-Hyperledger-Bevel.md:18]: Would you need additional assistance on this topic?

**Review by arsulegai — APPROVED** [review]

File: project-reports/2024/2024-Q3-Hyperledger-Bevel.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+<!-- -->
+
+-	Existing maintainers:
+	-	David Viejo (Kung Fu Software)
+	-	Aditya Joshi (Walmart)
+
+# Contributor Diversity
+[Dashboard]
+(https://insights.lfx.linuxfoundation.org/foundation/hyp/reports/contributors?project=bevel&repository=https:%2F%2Fgithub.com%2Fhyperledger%2Fbevel&routedFrom=Github)
```
Comment (denyeart) [inline project-reports/2024/2024-Q3-Hyperledger-Bevel.md]: This markdown is not formatting correctly as a link, probably because of the extra return.

**Review by denyeart — APPROVED** [review]

File: project-reports/2024/2024-Q3-Hyperledger-Bevel.md (hunk 2 above)
Comment (suvajit-sarkar) [reply project-reports/2024/2024-Q3-Hyperledger-Bevel.md:18]: Yes, please. Can connect to discuss

**Review by petermetz — APPROVED** [review]

File: project-reports/2024/2024-Q3-Hyperledger-Bevel.md (hunk 3 above)
Comment (suvajit-sarkar) [reply project-reports/2024/2024-Q3-Hyperledger-Bevel.md]: Have fixed this

File: project-reports/2024/2024-Q3-Hyperledger-Bevel.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+grand_parent: Project Updates
+has_children: false
+has_toc: false
+---
+
+# 2024 Q3 Hyperledger Bevel
+
+# Project Health
+
+[Hyperledger Bevel LFX Insights page](https://insights.lfx.linuxfoundation.org/foundation/hyp/overview?project=bevel&repository=&dateFilters=Last%20Year&dateRange=2023-01-01%20to%202023-12-31&compare=PP&granularity=month&hideBots=true)
```
Comment (tkuhrt) [inline project-reports/2024/2024-Q3-Hyperledger-Bevel.md]:
```suggestion
[Hyperledger Bevel LFX Insights page](https://insights.lfx.linuxfoundation.org/foundation/lf-decentralized-trust/overview?project=bevel&repository=&dateFilters=Last%20Year&dateRange=2023-01-01%20to%202023-12-31&compare=PP&granularity=month&hideBots=true)
```

File: project-reports/2024/2024-Q3-Hyperledger-Bevel.md [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+
+
+1. Getting code contributions for Hyperledger Bevel still remains a challenge.
+2. The Bevel project has received no external contributions; nearly all contributions for the last couple of releases have come from the Accenture team. Accenture will not be able to maintain the project without other organization or individual contributor help in maintaining.
+3. Community usage and interest are positive, as inferred from discussions and queries on the Discord channel.
+
+[Hyperledger Bevel Operator Fabric LFX Insights page](https://insights.lfx.linuxfoundation.org/foundation/hyp/overview?project=bevel&repository=https:%2F%2Fgithub.com%2Fhyperledger%2Fbevel-operator-fabric)
```
Comment (tkuhrt) [inline project-reports/2024/2024-Q3-Hyperledger-Bevel.md]:
```suggestion
[Hyperledger Bevel Operator Fabric LFX Insights page](https://insights.lfx.linuxfoundation.org/foundation/lf-decentralized-trust/overview?project=bevel&repository=https:%2F%2Fgithub.com%2Fhyperledger%2Fbevel-operator-fabric)
```

File: project-reports/2024/2024-Q3-Hyperledger-Bevel.md [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+
+Deployment using just helm commands with proxy/vault and without proxy/vault options for all DLT platform
+Ansible automations to use the new helm charts.
+
+# Current Plans
+
+- Continue with Q3 plan and release commitments.
+- Hyperledger meetup: Bevel workshop series remaining two sessions in September.
+- Explore cloud native vaults (AWS secret manager).
+- Collaborate with hyperledger-labs blockchain explorer and fabric-opertaions-console teams and plan to add deployment support on bevel.
```
Comment (tkuhrt) [inline project-reports/2024/2024-Q3-Hyperledger-Bevel.md]:
```suggestion
- Collaborate with hyperledger-labs blockchain explorer and fabric-operations-console teams and plan to add deployment support on bevel.
```

File: project-reports/2024/2024-Q3-Hyperledger-Bevel.md [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+<a href="https://github.com/hyperledger/bevel-operator-fabric/blob/main/MAINTAINERS.md" class="external-link" rel="nofollow">https://github.com/hyperledger/bevel-operator-fabric/blob/main/MAINTAINERS.md</a>.
+
+<!-- -->
+
+-	Existing maintainers:
+	-	David Viejo (Kung Fu Software)
+	-	Aditya Joshi (Walmart)
+
+# Contributor Diversity
+[Dashboard](https://insights.lfx.linuxfoundation.org/foundation/hyp/reports/contributors?project=bevel&repository=https:%2F%2Fgithub.com%2Fhyperledger%2Fbevel&routedFrom=Github)
```
Comment (tkuhrt) [inline project-reports/2024/2024-Q3-Hyperledger-Bevel.md]:
```suggestion
[Dashboard](https://insights.lfx.linuxfoundation.org/foundation/lf-decentralized-trust/reports/contributors?project=bevel&repository=https:%2F%2Fgithub.com%2Fhyperledger%2Fbevel&routedFrom=Github)
```

**Review by tkuhrt — CHANGES_REQUESTED** [review]

File: project-reports/2024/2024-Q3-Hyperledger-Bevel.md (hunk 2 above)
Comment (jimthematrix) [reply project-reports/2024/2024-Q3-Hyperledger-Bevel.md:18]: does this summary on external contributions take into account the `bevel-fabric-operator` component? Isn't that contributed by a different organization? or is that not considered under the Bevel umbrella, which if so would be a mistake.

**Review by jimthematrix — APPROVED** [review]
approved with a question

File: project-reports/2024/2024-Q3-Hyperledger-Bevel.md (hunk 2 above)
Comment (suvajit-sarkar) [reply project-reports/2024/2024-Q3-Hyperledger-Bevel.md:18]: the bevel and bevel-fabric-operator have separate code base/repo which runs independently, the comment here is for bevel

**Review by tkuhrt — APPROVED** [review]

File: project-reports/2024/2024-Q3-Hyperledger-Bevel.md (hunk 2 above)
Comment (arsulegai) [reply project-reports/2024/2024-Q3-Hyperledger-Bevel.md:18]: Would having an umbrella of projects help in reaching the goal? For instance, Cello is another project with kubernetes operators.

**Review by VRamakrishna — APPROVED** [review]

**Pull request merged**